Wangscape/tilegen/DebugTilesetID.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "ModuleGroupRole.h"
+#include "noise/ModuleID.h"
+
+namespace tilegen
+{
+
+struct DebugTilesetID
+{
+    ModuleGroupRole moduleGroupRole;
+    noise::ModuleID moduleID;
+    bool top;
+    bool left;
```

Comment:
They're used as `std::map` keys, so those values shouldn't be writeable. I've made them private.